frontend/lobby.js

```diff
@@ -40,17 +40,13 @@ socket.on("player-joined", (user) => {
   document.getElementById("player-count").innerText =
     oldPlayerCount + 1 + " / " + maxPlayers;
```

Comment:
oldPlayerCount is derived from the DOM and this is my fault and is not a secure way of getting the number of players. Changing it is a priority 3.